.circleci/config.yml

```diff
@@ -95,37 +110,56 @@ executors:
       PAYPAL_SANDBOX: true
       FXA_MX_RECORD_EXCLUSIONS: restmail.dev.lcip.org
 
-  # For anything that needs browsers for ui automation, but doesn't need a full stack.
-  # Perfect running smoke tests deployed code.
+  # Contains a pre-installed fxa stack and browsers for doing ui test
+  # automation. Perfect for running smoke tests against remote targets.
   smoke-test-executor:
     parameters:
       resource_class:
         type: string
         default: medium+
     resource_class: << parameters.resource_class >>
     docker:
-      - image: << pipeline.parameters.docker-repo >>:ci-base-browsers-latest
+      - image: mozilla/fxa-circleci:ci-functional-test-runner
+    environment:
+      NODE_ENV: development
+
 
 commands:
-  # Note: Cloning this way allows us to run a git fetch later on download the road. This type of
-  # clone operation will result in a .git/config with no user attached that is using a
-  # https://github.com/mozilla/fxa for the remote origin.
+
   git-clone:
+    # Note: Cloning this way allows us to run a git fetch & checkout later on download the road. This type of
+    # clone operation will result in a .git/config with no user attached that is using
+    # https://github.com/mozilla/fxa for the remote origin.
+    # The --depth argument is used to drastically cut down the initial download size. The size
+    # of the git folder is also much smaller, which ultimately results in smaller docker images.
     steps:
-      - run: git clone https://github.com/mozilla/fxa .
-      - run: git checkout << pipeline.git.revision >>
+      - run:
+          name: git clone
+          command: |
+            git clone https://github.com/mozilla/fxa . --depth=1
+            git fetch https://github.com/mozilla/fxa << pipeline.git.revision >> --depth=2
+            git checkout << pipeline.git.revision >>
 
-  # Note: Since we use a pre-populated base image that has a .git folder already present (see
-  # comment above), running a fetch and checkout saves a considerable amount of time  when compared
-  # to a circleci `- checkout` command.
   git-checkout:
+    # Note: Since we use a pre-populated base image that has a .git folder already present (see
+    # comment above), running a fetch and checkout saves a considerable amount of time when compared
+    # to a circleci `- checkout` command.
     steps:
       - run:
           name: Checkout Branch
           command: |
             set +x
-            git fetch
+            # If the PR is coming from a fork, add it as a remote, so the commit can be checked out.
```

Comment:
Fixes issues with forked PR pull requests.

Reply:
Very nice! We might have an issue open for this so could close another bug.